xds/internal/httpfilter/fault/fault_test.go

```diff
@@ -608,7 +608,7 @@ func (s) TestFaultInjection_MaxActiveFaults(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
-	streams := make(chan testpb.TestService_FullDuplexCallClient)
+	streams := make(chan testpb.TestService_FullDuplexCallClient, 5) // startStream() is called 5 times
```

Comment:
For this one, let's just change the `Fatalf` to an `Errorf`; I don't think it's a problem to continue running the rest of the test if an error happens.